lib/crypto/tasks.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from celeryutils import task
+
+from addons.models import Addon
+from lib.crypto.packaged import SigningError
+
+log = logging.getLogger('z.task')
+
+
+@task
+def sign_addons(addon_ids, **kw):
+    log.info('[%s] Signing addons.' % len(addon_ids))
+    addons = Addon.objects.filter(pk__in=addon_ids)
+    for addon in addons:
+        for version in addon.versions.all():
+            try:
+                version.sign_files()
+            except SigningError as e:
+                log.warning('Failed signing addon %s: %s.' % (addon.pk, e))
```

Comment:
You're right, I don't know why I didn't do that here, while I did it in the other PRs... my bad, i'll change that straight away